cou/cli.py

```diff
@@ -96,13 +100,15 @@ async def analyze_and_plan(
     :return: Generated analyses and upgrade plan.
     :rtype: tuple[Analysis, UpgradeStep]
     """
+    progress_indicator.start(f"Connecting to '{model_name or 'current-model'}' model...")
```

Comment:
Because after we creates it, it's already connected. So you will see nothing in (until model is not connected) and than you will se `Connecting to '<name>' model ...`. If we do so, there is no point of having and indicator.